eos-fix-external-appstream.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Rename or remove the outdated external appstream file
+ConditionPathExists=/var/cache/app-info/xmls/eos-extra.xml
```

Comment:
The script handles `eos-extra.xml.gz`, but this is only running when `eos-extra.xml` exists.